### DISCUSSION TITLE
gdbm Error: Failure while executing; patch -g 0 -f -p0 exited with 1.

### DISCUSSION THREAD

**Discussion by pulberg** [Homebrew/discussions#2184] [Linux] [answered]
==> Upgrading gdbm
  1.20 -> 1.21 

tar --extract --no-same-owner --file /home/pulberg/.cache/Homebrew/downloads/860a85eeb67ee18de3493ae713bce14b2252e342be679887bce3b7cb2c15dae6--gdbm-1.21.tar.gz --directory /tmp/d20210923-162217-14rtmce
cp -pR /tmp/d20210923-162217-14rtmce/gdbm-1.21/. /tmp/gdbm-20210923-162217-1erwigp/gdbm-1.21
chmod -Rf +w /tmp/d20210923-162217-14rtmce
==> Patching
Error: Failure while executing; `patch -g 0 -f -p0` exited with 1. Here's the output:
patching file src/gdbmshell.c
Hunk #1 FAILED at 1010.
1 out of 1 hunk FAILED -- saving rejects to file src/gdbmshell.c.rej

**Comment by FnControlOption** [comment, marked as answer]
Submitted a fix https://github.com/Homebrew/homebrew-core/pull/85852
